ra-tls/Makefile

```diff
@@ -83,17 +83,14 @@ $(LIBDIR)/libra-challenger.a: $(LIBDIR) $(LIBDIR)/libwolfssl.a $(SGX_RA_TLS)/ra.
 # -DFP_MAX_BITS=8192 required for RSA keys > 2048 bits to work
 $(LIBDIR)/libwolfssl.sgx.static.lib.a: $(LIBDIR) $(LIBDIR)/libwolfssl.a
 	cd wolfssl/IDE/LINUX-SGX && \
-	  make -f sgx_t_static.mk CFLAGS="-DUSER_TIME -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_KEY_GEN -DWOLFSSL_CERT_GEN -DWOLFSSL_CERT_EXT -DFP_MAX_BITS=8192" && \
+	  make -f sgx_t_static.mk CFLAGS="-DUSER_TIME -DWOLFSSL_SGX_ATTESTATION -DWOLFSSL_KEY_GEN -DWOLFSSL_CERT_GEN -DWOLFSSL_CERT_EXT -DFP_MAX_BITS=8192 -DKEEP_PEER_CERT" && \
 	  cp -f libwolfssl.sgx.static.lib.a "$@"
 
-$(LIBDIR)/libsgx_ra_tls_wolfssl.a: $(LIBDIR)
-# Previous Makefile compiles these .o files with incorrect C flags
-# Don't disturb the build of libsgx_ra_tls_wolfssl.a
-	rm -f $(SGX_RA_TLS)/wolfssl-ra-challenger.o $(SGX_RA_TLS)/wolfssl-ra.o $(SGX_RA_TLS)/ra-challenger.o $(SGX_RA_TLS)/ias_sign_ca_cert.o
-	$(MAKE) -C $(SGX_RA_TLS) && \
-	  mv -f $(SGX_RA_TLS)/libsgx_ra_tls_wolfssl.a "$@"
-# Don't disturb the build of libra-challenger.a
-	rm -f $(SGX_RA_TLS)/wolfssl-ra-challenger.o $(SGX_RA_TLS)/wolfssl-ra.o $(SGX_RA_TLS)/ra-challenger.o $(SGX_RA_TLS)/ias_sign_ca_cert.o
+$(LIBDIR)/libenclave-tls.so: $(LIBDIR) $(LIBDIR)/libcurl-wolfssl.a $(LIBDIR)/libwolfssl.a
```

Comment:
Is it possible to make libcurl-wolfssl.so and libwolfssl.so available for untrusted libenclave-tls.so?

Reply:
It can make libcurl-wolfssl.so and libwolfssl.so available.